GameServerLib/Logic/GameObjects/Champion.cs

```diff
@@ -177,22 +177,38 @@ public int getTeamSize()
                 var p = config.Players[playerIndex];
                 playerTeam = p.Team;
             }
-            var x = 0;
-            var y = 0;
+
+            var toReturn = new Vector2();
```

Comment:
How about
```CS
var spawnsByTeam = new Dictionary<string, (whatever type the spawns are)> = new Dictionary<string, x>
{
    {"blue": config.MapSpawns.Blue},
    {"purple": config.MapSpawns.Purple}
}
// Check for key existance here
var spawns = spawnsByTeam[playerTeam.ToLower()];
return new Vector2(
    spawns[teamSize - 1].GetXForPlayer((int)_playerTeamSpecialId),
    spawns[teamSize - 1].GetYForPlayer((int)_playerTeamSpecialId)
);
// Even better than above ^ would be to add a `.GetForPlayer` method to the spawns that returns a Vector2,
// instead of having two different function for fetching floats.
```